### DISCUSSION TITLE
Addition of DoVi demux + injecting + editing + parser in Copy/Mux option in staxrip.

### DISCUSSION THREAD

**Discussion by STENCv2** [staxrip/staxrip#1341] [Ideas]
Hi guys just wanted to ask you all whether a addition of a feature could be possible in copy/mux mode of staxrip where i would be able to inject demuxed dovi layer recent updates enable to add it automatically but i use chunk mode to encode where in i cannot allow DoVi to be added and hence i remove it, so i was wondering whether in Copy/Mux mode i will be able to add this dovi layer from the source to the encode???

**Comment by STENCv2** [comment]
No responses from admins! :(

**Comment by madey83** [comment]
> Hi guys just wanted to ask you all whether a addition of a feature could be possible in copy/mux mode of staxrip where i would be able to inject demuxed dovi layer recent updates enable to add it automatically but i use chunk mode to encode where in i cannot allow DoVi to be added and hence i remove it, so i was wondering whether in Copy/Mux mode i will be able to add this dovi layer from the source to the encode???

try this: https://github.com/R3S3t9999/DoVi_Scripts

**Comment by jessielw** [comment]
I'm not a developer for StaxRip but you can encode your video, simply use dovi tool CLI to inject into the hevc in the temp folder (keep same name) and remux with staxrip by adding the SRIP file and pressing add job, reuse audio, reuse video.
